app/javascript/hyacinth_v1/components/shared/TopNavbar.js

```diff
@@ -57,13 +57,13 @@ function TopNavbar(props) {
 
             <NavDropdown.Divider />
 
-            <LinkContainer to="/batch_exports">
```

Comment:
🤔

Reply:
?

Reply:
I switched the order of the links to match Hyacinth 2.  We accidentally had them reversed here.